draft-ietf-core-object-security.md

```diff
@@ -462,11 +462,35 @@ The COSE Object SHALL be a COSE_Encrypt0 object with fields defined as follows
    * The "Partial IV" parameter. The value is set to the Sender Sequence Number. All leading zeroes SHALL be removed when encoding the Partial IV. The value 0 encodes to the byte string 0x00. This parameter SHALL be present in requests. In case of Observe ({{observe}}) the Partial IV SHALL be present in responses, and otherwise the Partial IV SHOULD NOT be present in responses. (A non-Observe example where the Partial IV is included in a response is provided in {{reboot-replay}}.)
 
    * The "kid" parameter. The value is set to the Sender ID. This parameter SHALL be present in requests and SHOULD NOT be present in responses. (An example where the Sender ID is included in a response is the extension of OSCORE to group communication {{I-D.tiloca-core-multicast-oscoap}}.)
+   
+   * Optionally, a "context hint" parameter as defined in {{context-hint}}. This parameter MAY be present in requests and SHALL NOT be present in responses.
 
```

Comment:
I think that perhaps the term "kid context" might be a better term.  I dislike hint because it implies that it is a helper not a restraint.

Reply:
@gselander you were the one coming up with the term "Context Hint". Would you be happy with this change? I personally don't mind, "kid context" is fine.